kumascript/macros/LearnSidebar.ejs

```diff
@@ -168,6 +168,19 @@ var text = mdn.localStringMap({
       'Accessible_multimedia' : 'Accessible multimedia',
       'Mobile_accessibility' : 'Mobile accessibility',
       'Assessment_Accessibility_troubleshooting' : 'Assessment: Accessibility troubleshooting',
+    'Performance' : 'Performance — Making websites fast and responsive',
+      'Performance_guides' : 'Performance guides',
+        'Performance_overview' : 'Performance overview',
+        'The_why_of_performance' : 'The why of performance',
+        'What_is_performance' : 'What is performance',
+        'How_do_users_perceive_performance' : 'How do users perceive performance',
+        'Measuring_performance' : 'Measuring performance',
+        'Multimedia_images' : 'Multimedia images',
+        'Multimedia_video' : 'Multimedia video',
+        'JavaScript_performance_best_practices' : 'JavaScript performance best practices',
+        'HTML_performance_features' : 'HTML performance features',
+        'CSS_performance_features' : 'CSS performance features',
+        'The_business_case_for_performance' : 'The business case for performance',
```

Comment:
There are a few cases here where the link text is not ideal, for example:

> What is performance
> How do users perceive performance

... questions without a question mark.

> Multimedia images
> Multimedia video

...these don't really make sense, a "Multimedia image" isn't a thing. Either "Multimedia: images" or "Images".

Overall and especially if we want to go ahead with https://github.com/mdn/yari/pull/8132, I'd suggest using the titles of pages as the sidebar link text, unless there's a good reason not to.